<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>530</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">SwapTool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/ui/alogic_logo.ico</normaloff>:/ui/alogic_logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>444</width>
     <height>491</height>
    </rect>
   </property>
   <widget class="QPlainTextEdit" name="statusText">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>440</width>
      <height>131</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="228">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="179">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="228">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="179">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="backgroundVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="memoryDetailsBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>440</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Memory Overview</string>
    </property>
    <widget class="QPushButton" name="refreshButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QLabel" name="memLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>RAM:</string>
     </property>
    </widget>
    <widget class="QLabel" name="swapLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Swap:</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalRamLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="usedRamLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="freeRamLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalSwapLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="usedSwapLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="freeSwapLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="ramProgressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>400</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="swapProgressBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>400</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="infoPartitionLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Swap Partition:</string>
     </property>
    </widget>
    <widget class="QLabel" name="infoFileLabel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>210</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Swap File:</string>
     </property>
    </widget>
    <widget class="QLabel" name="swapFileInputLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>253</y>
       <width>110</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Swap size:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="swapFileInput">
     <property name="geometry">
      <rect>
       <x>143</x>
       <y>256</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sfileInputLabel">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>258</y>
       <width>50</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MB</string>
     </property>
    </widget>
    <widget class="QPushButton" name="applyButton">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>300</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QLabel" name="partitionSizeLabel">
     <property name="geometry">
      <rect>
       <x>121</x>
       <y>215</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileSizeLabel">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>214</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
